assets/templates/partials/header.tmpl

```diff
@@ -248,4 +248,4 @@
             </div>
         </div>
     {{end}}
-</header>
+</header>
```

Comment:
No empty line at the end of the file

This is the setting in vs code
<img width="498" alt="Screenshot 2021-07-28 at 15 15 06" src="https://user-images.githubusercontent.com/19624419/127338083-b9530cfb-58f6-403c-90c0-911816c67c42.png">